### ISSUE TITLE
Create a trial variation with no storage dependency 

### ISSUE THREAD

**Issue by dariagrigoriu** [Azure/azure-webjobs-sdk-script#1282]
Create a trial Function App variation with no storage dependency. The trial variation can be scoped to HTTP trigger and single instance functions.

**Comment by lindydonna** [comment]
What is the goal?  Seems like this is more of a portal request than script host.

**Comment by mathewc** [comment]
Yes, I have similar questions. Please clarify the ask by adding details.

**Comment by christopheranderson** [comment]
Need to identify improvements/bugs that are related to removing Storage dependency. @dariagrigoriu to coordinate investigation.

**Comment by fabiocav** [comment]
The initial scope of this work, from a runtime perspective, is to ensure the lease logic doesn't have any issues and treat the host as the primary when the storage account is not available.

We'd likely need an item to track any related work required on the scale controller to ensure we're not scaling when storage is not available.
